Sources/Action/UIKitExtensions/UIBarButtonItem+Action.swift

```diff
@@ -37,5 +38,28 @@ public extension Reactive where Base: UIBarButtonItem {
             }
         }
     }
+    public func bindToAction <Input,Output>(_ action:Action<Input,Output>?,_ inputTransform: @escaping (Base) -> (Input))   {
```

Comment:
Ah, that makes sense, though our current `action` properties aren't optional and we've never had a request to make them optional. As you [noted](https://github.com/RxSwiftCommunity/Action/pull/72#issuecomment-270722051), users could get around this limitation by binding a new, empty action. I'd prefer to keep things non-optional if we can, but if anyone feels strongly the other way, we can chat about it 👍